### PR TITLE
Change: scope app styling with css baseline container

### PR DESCRIPTION
**Pull request by defunctzombie**
**User-Facing Changes**
None

**Description**
Rather than setting styles globally on elements, scope styling
to a top level app container. The CssBaseline component should be
used as this top level app container.

### CODE REVIEW

File: desktop/renderer/index.tsx [hunk 1]
```diff
@@ -48,6 +49,21 @@ if (!rootEl) {
...
+    width: "100%",
+    display: "flex",
+    flexDirection: "column",
+    position: "relative",
+    flex: "1 1 100%",
+    outline: "none",
+    overflow: "hidden",
+  },
+});
+
```
Comment (2metres) [inline desktop/renderer/index.tsx:66]: We should probably consider name spacing our root selector something like `#foxglove-root`

**Review by 2metres — CHANGES_REQUESTED** [review]
The typography is currently broken in chromatic and the background is now included

File: desktop/renderer/index.tsx (hunk 1 above)
Comment (defunctzombie) [reply desktop/renderer/index.tsx:66]: Any benefit to this since we don't reference it in studio-base?

**Comment by defunctzombie** [comment]
TextContent has a line height of 1.65 - this causes too much space between lines.

**Pull request merged**